pandas/tests/frame/indexing/test_xs.py

```diff
@@ -0,0 +1,95 @@
+import re
+
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import DataFrame, Series
+import pandas._testing as tm
+
+from pandas.tseries.offsets import BDay
+
+
+class TestXS:
+    def test_xs(self, float_frame, datetime_frame):
```

Comment:
no big deal, but happy to have no classes if this is a specific test file (like you have it now).